interface/modules/custom_modules/oe-module-comlink-telehealth/tests/bootstrap.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * phpunit bootstrap file
+ *
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2022 Comlink Inc <https://comlinkinc.com/>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
```

Comment:
is this something that is only run via command line. If so, would make sense to lock it down with (just to avoid somebody try to brute-force runs through globals.php with every get/post variable known to man):
(if this is case, then we should also do same in the main `tests/bootstrap.php` script)
```
if (php_sapi_name() !== 'cli') {
exit:
}
```

Reply:
I'll add it to both the main unit tests and the module unit tests.

Reply:
fixed